spec/commands/generate-spec.js

```diff
@@ -1,26 +1,24 @@
-import proxyquire from 'proxyquire';
-import sinon from 'sinon';
-import { expect } from 'chai';
+import { createMigrationFile } from '../../lib/migrations/migration-files';
+import generate from "../../lib/commands/generate";
 
-import createMockLogger from '../helpers/create-mock-logger';
+jest.mock('../../lib/database/database-client');
 
-describe("generate", () => {
-  let generate, createMigrationFileStub, migrationsDirectory;
+// Suppress the logging.
+jest.mock('../../lib/utilities/logger');
```

Comment:
Can we do this is a test helper file? You can see UI for how to add one to the Jest config.

Reply:
We can't do that because we need to actually execute the logger code in the logger tests, while mocking process.stdout.write()